canonical-web-and-design/ubuntu.com
# Rebuild openstack/features page

## Done

-  Rebuild `openstack/features` 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/features
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#) 


## Issue / Card

Fixes [#3872](https://github.com/canonical-web-and-design/web-squad/issues/3872)
